README.md

```diff
@@ -45,17 +45,26 @@ All members that have been inducted for standard allowance as per the [Fellowshi
 All members must serve a minimum period of service of 12 months at their current rank before seeking promotion to a higher rank. 
 
 
+## Timelines
+
+Members can individually monitor the progress of their demotion period on the [Core Fellowship UI](https://collectives.subsquare.io/fellowship/core) provided by Subsquare. Members will soon be able to set up an ical feed to set reminders and get updates for their membership status.
+
+It is recommended that members submit their evidence for review (via GitHub) and on-chain (via Subsquare) as per the following deadlines:
+- For retentions: no later than **30 days** prior to the end of the demotion period
+- For promotions: no later than **40 days** prior to the end of the demotion period
+
+
 ## Process
 
 The process for submitting Evidences is open to all existing Fellowship members (i.e Rank I to IX). Anyone may provide comments on submitted Evidences.
 
 To submit an Evidence, follow these steps:
-  * Fork the `Evidences` repository
-  * Create a new folder in the `evidence` folder and rename it to match your Github username
-  * Copy the `0000-evidence-template.md` file into the new folder and rename it to match the title of your request
+  * Fork the `Evidences` repository.
+  * Create a new folder in the `evidence` folder and rename it to match your Github username.
+  * Copy the `0000-evidence-template.md` file into the new folder and rename it to match the title of your request.
   * Fill out the Evidence template and open a PR.
-  * Announce the evidence to the fellowship and wait at least one week.
-  * If there are no major push backs by the fellowship, submit the evidence on-chain with the `fellowshipCore.submitEvidence(wish, evidence)` call on the [Polkadot Collectives chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fsys.ibp.network%2Fcollectives-polkadot#/extrinsics/decode/0x3f0700081230), where *wish* is the nature of the request (Retention or Promotion) and *evidence* is the blake2-256 hash of the raw evidence text.
+  * Announce the evidence to the Fellowship and wait at least one week.
+  * If there are no major pushbacks by the Fellowship, submit the evidence on-chain via the [Core Fellowship UI](https://collectives.subsquare.io/fellowship/core) provided by Subsquare.
```

Comment:
In the future we should be able to use subsquare for the entire process. However, I'm not sure we are already there yet.

Reply:
The main issue is that sometimes IPFS links don't work (due to browser/OS compatibility issues) and you can't access/read the actual evidence. 
Also, without an open archive for evidences where the discussions/reviews can happen, everything is going to be scattered, leading to low quality/incomplete evidence reports, which would defeat the purpose of having a standardised procedure in the end.

Reply:
> In the future we should be able to use subsquare for the entire process. However, I'm not sure we are already there yet.

The main issue is that sometimes IPFS links don't work (due to browser/OS compatibility issues) and you can't access/read the actual evidence. 
Also, without an open archive for evidences where the discussions/reviews can happen, everything is going to be scattered, leading to low quality/incomplete evidence reports, which would defeat the purpose of having a standardised procedure in the end.

Reply:
> The main issue is that sometimes IPFS links don't work (due to browser/OS compatibility issues) and you can't access/read the actual evidence.

These are things that subsquare needs to solve. 

> Also, without an open archive for evidences where the discussions/reviews can happen, everything is going to be scattered, leading to low quality/incomplete evidence reports, which would defeat the purpose of having a standardised procedure in the end.

Another thing we need then talk to them to have some standarized form or whatever. The current approach of doing a PR and putting it onto ipfs is a little bit stupid. We should do one approach and that correctly. However, we should do this over time.